README.md

```diff
@@ -0,0 +1,36 @@
+# MSBuild plugin
+
+[![Build Status](https://ci.jenkins.io/job/Plugins/job/msbuild-plugin/job/master/badge/icon)](https://ci.jenkins.io/job/Plugins/job/msbuild-plugin/job/master/)
+
+This plugin allows you to use MSBuild to build .NET and Visual Studio
+projects.
+
+## Usage
+
+To use this plugin, specify the location directory of MSBuild.exe on
+Jenkin's configuration page. The MSBuild executable is usually
+situated in a subfolder of C:\\WINDOWS\\Microsoft.NET\\Framework. The
+Visual Studio 2017 Community version of "msbuild.exe" is located in
+"C:\\Program Files (x86)\\Microsoft Visual
+Studio\\2017\\Community\\MSBuild\\15.0\\Bin\\" If you have multiple
```

Comment:
`"C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Community\\MSBuild\\15.0\\Bin\\"`

Reply:
Not sure what you're suggesting here?

Reply:
I was trying to suggest https://github.com/jenkinsci/msbuild-plugin/pull/39#discussion_r353014408 😅